lynx/patterns/link-in-bio-purple-background.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Title: Link in Bio with purple background
+ * Slug: lynx/link-in-bio-purple-background
+ * Inserter: yes
+ * Categories: featured, link-in-bio
+ */
+?>
+
+<!-- wp:cover {"customOverlayColor":"#7749f9","minHeight":100,"minHeightUnit":"vh","align":"full","style":{"spacing":{"padding":{"top":"8vh","right":"5vh","bottom":"8vh","left":"5vh"}}}} -->
```

Comment:
Inserting this pattern results in a strange layout:

<img width="1624" alt="Screen Shot 2022-08-18 at 2 50 33 PM" src="https://user-images.githubusercontent.com/5375500/185472485-8aaf6218-ec76-40f5-a679-443e845b4189.png">

Reply:
Yes, I noticed this too. I think it's an editor issue: https://github.com/WordPress/gutenberg/issues/34909

Reply:
This particular pattern's setup is prooving too complex, so I'll likely be removing it, at least from the initial commit.